frontend/lib/widgets/nav_drawer.dart

```diff
@@ -0,0 +1,139 @@
+                ),
+            const SizedBox(
+              height: 16,
+            ),
+            buildMenuItem(
+                text: 'My Events',
+                icon: Icons.event,
+                onClicked: () => selectedItem(context, 2)),
+            const SizedBox(
+              height: 16,
+            ),
+            buildMenuItem(
+              text: 'Events I\'m Attending',
+              icon: Icons.event_available,
+              // onClicked: () => selectedItem(context, 3)
+            ),
+            // divider
+            const SizedBox(
+              height: 24,
+            ),
+            Divider(
+              color: Colors.black54,
+            ),
+            const SizedBox(
+              height: 16,
+            ),
+            // profile and settings
+            const SizedBox(
+              height: 16,
+            ),
+            buildMenuItem(
+              text: 'Profile',
+              icon: Icons.person,
+              // onClicked: () => selectedItem(context, 4)
+            ),
+            const SizedBox(
+              height: 16,
+            ),
+            buildMenuItem(
+              text: 'Settings',
+              icon: Icons.settings,
+              // onClicked: () => selectedItem(context, 5)
+            ),
+            const SizedBox(
+              height: 16,
+            ),
+            buildMenuItem(
+                text: 'Log out',
+                icon: Icons.exit_to_app,
+                onClicked: () => selectedItem(context, 6)),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget buildMenuItem(
+      {required String text, required IconData icon, VoidCallback? onClicked}) {
+    final color = Colors.black54;
+    final hoverColor = Colors.red;
+
+    return ListTile(
+      leading: Icon(icon, color: color),
+      title: Text(text, style: TextStyle(color: color)),
+      hoverColor: hoverColor,
+      onTap: onClicked,
+    );
+  }
+
+  void selectedItem(BuildContext context, int index) {
+    Navigator.of(context).pop(); // before switching pages, close the menu
+    switch (index) {
+      case 0:
+        print('Browse events clicked');
+        Navigator.of(context)
+            .push(MaterialPageRoute(builder: (context) => BrowseEvents()));
+        break;
+      case 1:
+        Navigator.of(context)
+            .push(MaterialPageRoute(builder: (context) => DummyPage()));
+        break;
+      case 2:
+        Navigator.of(context)
+            .push(MaterialPageRoute(builder: (context) => EventPg()));
+        break;
+      // case 3:
+      //   Navigator.of(context).push(MaterialPageRoute(builder: (context) => YourPageName()));
+      //   break;
+      // case 4:
+      //   Navigator.of(context).push(MaterialPageRoute(builder: (context) => YourPageName()));
+      //   break;
+      // case 5:
+      //   Navigator.of(context).push(MaterialPageRoute(builder: (context) => YourPageName()));
+      //   break;
```

Comment:
Commented code shouldn't be in the master branch :)

Reply:
Uncommented this block in commit 029e4c6f432a732625dbf521275750643f5e2128